src/Network/Minio/Data.hs

```diff
@@ -184,21 +184,31 @@ type Object = Text
 -- TODO: This could be a Sum Type with all defined regions for AWS.
 type Region = Text
 
--- | A type alias to represent an Entity-Tag returned by S3-compatible
--- APIs.
+-- | A type alias to represent an Entity-Tag returned by S3-compatible APIs.
 type ETag = Text
 
 -- |
 -- Data type represents various options specified for PutObject call.
 -- To specify PutObject options use the poo* accessors.
 data PutObjectOptions = PutObjectOptions {
+  -- | A standard MIME type describing the format of the contents.
     pooContentType        :: Maybe Text
+  -- | Specifies what content encodings have been applied to the object and thus
```

Comment:
I think we should describe what this field is meant for and need not explain how Content-Encoding header works with Content-Type.

Reply:
That is what its meant for @krisis